<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_GUIClass</class>
 <widget class="QMainWindow" name="client_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>511</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>IP_ADRESS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PORT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>56</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>USER_ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PASSWORD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>SIGN IN</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:12pt;&quot;&gt;Please Write Correct Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QTextEdit" name="cmd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>280</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>QUIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="portButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PORT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pasvButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PASV</string>
      </property>
     </widget>
     <widget class="QListWidget" name="clientList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>201</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="serverList">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>201</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>REFRESH</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="address">
      <property name="geometry">
       <rect>
        <x>233</x>
        <y>60</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="EXportButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CHANGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="typeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TYPE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="systButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SYST</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mkdButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>160</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>MKD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="rmdButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>200</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RMD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rnButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RN</string>
      </property>
     </widget>
     <zorder>cmd</zorder>
     <zorder>quitButton</zorder>
     <zorder>portButton</zorder>
     <zorder>pasvButton</zorder>
     <zorder>serverList</zorder>
     <zorder>refreshButton</zorder>
     <zorder>clientList</zorder>
     <zorder>address</zorder>
     <zorder>EXportButton</zorder>
     <zorder>typeButton</zorder>
     <zorder>systButton</zorder>
     <zorder>mkdButton</zorder>
     <zorder>inputEdit</zorder>
     <zorder>rmdButton</zorder>
     <zorder>rnButton</zorder>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="client_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
